src/main/java/com/iridium/iridiumskyblock/gui/BiomeCategoryGUI.java

```diff
@@ -61,7 +62,7 @@ public void addContent(Inventory inventory) {
 
     private List<Placeholder> getBiomeLorePlaceholders(Biomes.BiomeItem item) {
         List<Placeholder> placeholders = new ArrayList<>(Arrays.asList(
-                new Placeholder("amount", IridiumSkyblock.getInstance().getBiomeManager().formatPrice(item.defaultAmount)),
+                new Placeholder("minLevel", String.valueOf(item.minLevel)),
```

Comment:
Do we not need the Amount placeholder anymore?

Reply:
yes, this allows us to fetch the price of the schematic from the config directly without having to set the lore manually every time